City-of-Helsinki/berth-reservations
# Contracts integration changes

## Description :sparkles:

Adds a script to generate contracts for leases currently missing this data in staging and prod. Related slack discussion on why this is needed: https://helsinkicity.slack.com/archives/CBA1QHTHN/p1607072913015400

Additionally updates the berth contract files and makes copies of contract data when renewing a lease.

### Manual testing :construction_worker_man:

Testing the script:

- Cross fingers
- Run the script
- Verify all leases in `DRAFTED` and `OFFERED` states now have contracts
- Depending on the amount of leases, this step might take a while as the documents are prepared in the third party service

If errors occur, figure out next steps from the logs for the ones that failed.
